src/server/rollbar.js

```diff
@@ -0,0 +1,233 @@
+var util = require('util');
+
+var Client = require('../rollbar');
+var _ = require('../utility');
+var logger = require('./logger');
+
+var transforms = require('./transforms');
+var predicates = require('./predicates');
+
+function Rollbar(accessToken, options, client) {
+  options = options || {};
+  options.accessToken = accessToken;
+  this.context = 'server';
+  this.initialize(options, client);
+}
+
+Rollbar.prototype.global = function(options) {
+  this.client.global(options);
+};
+
+Rollbar.prototype.configure = function(options) {
+  this.options = _.extend(true, {}, this.options, options);
+  this.client.configure(options);
+};
+
+Rollbar.prototype.initialize = function(options, client) {
```

Comment:
Can we just move this into the constructor? When would `initialize()` be called externally?